operator/values.yaml

```
@@ -0,0 +1,39 @@
+
+roleFinalizersVerbs:
+  - update
+
+roleSubjectaccessreviewsVerbs:
+  - create
+
+roleTokenreviewsVerbs:
+  - create
+
+namespace:
+  default
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)